src/core/services/utility.js

```diff
@@ -26,6 +28,9 @@ export {
 	isUndefined,
 	clone,
 	debounce,
+	merge,
+	yes,
```

Comment:
How about `truthy`|`falsy` or even `true` | `false` ?
Similar naming is used in assertion libs